routes/tripRouter.js

```diff
@@ -1,46 +1,154 @@
 const express = require('express');
 const tripRouter = express.Router();
 const TripService = require('../services/tripService');
-const userService = require('../services/userService')
+const userService = require('../services/userService');
+const BagService = require("../services/bagService");
+const ItemService = require('../services/itemService');
 
 tripRouter.post('/', (req, res, next) => {
-    const { name, country, city, departure_date, return_date, user_uid } = req.body;
+    const {
+        name,
+        country,
+        city,
+        departure_date,
+        return_date,
+        user_uid
+    } = req.body;
     userService.readUserByUid(user_uid)
-    .then(user => {
-        if (user) {
-            return TripService.create(name, country, city, departure_date, return_date, user.id) 
-        } else {
-            return TripService.create(name, country, city, departure_date, return_date, null)
+        .then(user => {
+            if (user) {
+                return TripService.create(name, country, city, departure_date, return_date, user.id)
+            } else {
+                return TripService.create(name, country, city, departure_date, return_date, null)
+            }
+        })
+        .then(({
+            id
+        }) => {
+            res.status(200);
+            res.json({
+                id
+            });
+        })
+        .catch(err => {
+            console.log(err)
+            next(err);
+        })
+
+});
+
+//trip creation v2, handles every aspect of the trip creation from one route
+tripRouter.post('/v2', async (req, res, next) => {
```

Comment:
IDK if /v2 is the best pathname for a query.

Reply:
Any suggestions?